mail/job.go

```diff
@@ -21,5 +23,49 @@ func (r *SendMailJob) Signature() string {
 
 // Handle Execute the job.
 func (r *SendMailJob) Handle(args ...any) error {
-	return SendMail(r.config, args[0].(string), args[1].(string), args[2].(string), args[3].(string), args[4].([]string), args[5].([]string), args[6].([]string), args[7].([]string))
+	if len(args) != 8 {
+		return fmt.Errorf("expected 8 arguments, got %d", len(args))
+	}
+
+	from, ok := args[0].(string)
+	if !ok {
+		return fmt.Errorf("argument 0 should be of type string")
```

Comment:
The action of arguments, please replace `argument *` with the real meaning.

<img width="998" alt="image" src="https://github.com/user-attachments/assets/51f94539-fe3a-4a21-b7ac-217179e567c6">